src/pages/Home/Home.js

```diff
@@ -59,8 +59,10 @@ function Home({
 
   useEffect(() => {
     setResultSummary(
-      `${templates.length || 0} Templates ${
-        categoryValue !== categoryOptions[0] || searchTerm !== "" ? "Found" : ""
+      `${
+        categoryValue !== categoryOptions[0] || searchTerm !== ''

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/NUKSI911/FormPlus/pullRequest?prid=7552120)
```suggestion
        categoryValue !== categoryOptions[0] || searchTerm !== ""
```